package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "error-tracker",
+  "version": "0.0.0",
```

Comment:
Add description and license info here. https://docs.npmjs.com/files/package.json